### PR TITLE
Mythic multiselect

### PR DESCRIPTION
**Pull request by emeeks**
Adds a simple myth for multiselecting cells exposing three functions: `selectCell`, `unselectCell` and `clearSelectedCells` which you can use to add or remove cell ids from a list of cell ids.

Includes tests but is not wired up to any functionality.

### CODE REVIEW

**Review by willingc — APPROVED** [review]
LGTM. Thanks @emeeks. I will leave this for the amazing @captainsafia to merge. Thanks again for all the help yesterday Safia.

**Comment by willingc** [comment]
Not sure why the test is failing.

File: packages/mythic-multiselect/__tests__/multiselect.spec.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    );
+
+    const nextState = multiselect.rootReducer(
+      state,
+      unselectCell.create({
+        id: "test2",
+      })
+    );
+
+    expect(nextState.selectedCells.size).toBeLessThan(state.selectedCells.size);
```
Comment (willingc) [inline packages/mythic-multiselect/__tests__/multiselect.spec.ts]: Will there always be at least 1 in the `size`?

Comment (emeeks) [reply packages/mythic-multiselect/__tests__/multiselect.spec.ts]: No, the test was miswritten slightly and got fixed in the later commits.

**Pull request merged**